pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -388,6 +397,24 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
 
     /* ========== RESTRICTED FUNCTIONS ========== */
 
+    /**
+     * @notice Allows a rewards distributor to deposit more tokens to be distributed as rewards
+     * @param rewardsToken - the token to deposit into staking contract for distribution
+     * @param reward - the amount of tokens to deposit
+     */
+    function startScheduledReward(
+        IERC20 pool,
+        IERC20 rewardsToken,
+        uint256 reward,
+        address rewarder
+    ) external override updateReward(pool, address(0)) {
+        require(
+            rewarder == msg.sender || msg.sender == rewardsScheduler,
+            "Rewarder must be sender, or rewards scheduler"
+        );
+        _startReward(pool, rewardsToken, reward, rewarder);
+    }
```

Comment:
Do actually need this function at all? It seems like we should just call `notifyRewardAmount` directly from the scheduler.

Reply:
I thought it was more clear to have 2 functions - `startScheduledReward` for the rewardScheduler, that takes a rewarder parameter, and another function `notifyRewardAmount' for self-scheduled rewards (which omits the rewarder parameter, and uses msg.sender).

If we want to require rewarders to pass their own address for that parameter, we can combine into 1 function.

Reply:
We _could_ have such a thing, but if so I'd make it so `startSchedulerReward` was only callable by the scheduler - I found the fact that rewarders could use either very confusing.

I guess there might be benefits in keeping `notify` as-is in that we'd remain interface-compatible with the original implementation? But I'm not sure.